spec/system/admin/configuration/zones_spec.rb

```diff
@@ -6,6 +6,10 @@
   include AuthenticationHelper
   include WebHelper
 
+  before do
+    Spree::Zone.delete_all
+  end
```

Comment:
How come we have zones in the database? I thought we have only countries and states by default.

Reply:
They are added during seeding : https://github.com/openfoodfoundation/openfoodnetwork/blob/master/db/default/zones.rb
Thanks for making me look into this, I found out that seeding was broken after my changes.